Forms/Access SVN - Options.ACF

```diff
@@ -2496,6 +2580,18 @@ Private Sub cmdFolderDialog_Click()
   Call txtOutputLocation_AfterUpdate
 End Sub
 
+Private Sub cmdRemoveGUIDs_Click()
+  
+  DoCmd.Hourglass True
+  If CleanFormsReports(Me.txtOutputLocation.Value) Then
+    SetConfigValue "Remove GUIDs", True, appendSlash(Me.txtOutputLocation.Value) & gstrConfigFilename
+  End If

```

Comment:
Should the user be warned if this fails, as in the reencoding?

Reply:
Done